cmd/performance-profile-creator/README.md

```diff
@@ -0,0 +1,55 @@
+# Performance Profile Creator (PPC)
+
+A tool to automate the process of creating Performance Profile using the user supplied profile parameters.
+
+## Software Components
+
+1. A CLI tool part of the PAO image
+1. A wrapper script <!--(TODO: Add link to where the wrapper script is placed) --> automates the access to PPC tool. It pulls the PAO image, supplies arguments and runs the tool inside Performance Addon Operator container with performance-profile-creator entrypoint.
+
+## Flow
+1. The user supplies the command line args to the tool by using a wrapper script.
+1. The wrapper triggers the must-gather tool.
```

Comment:
it is correct that all the bullet points start with `1.` ?

Reply:
Yep, I think it should be fine as they are rendered as 1, 2, 3 etc., I learnt this recently while writing NFD documentation. Take a look [here](https://guides.github.com/features/mastering-markdown/).